lib/Language/Haskell/Homplexity/Message.hs

```diff
@@ -27,9 +28,15 @@ import Data.Monoid
 import Data.Semigroup
 #endif
 import Data.Sequence                            as Seq
-import Language.Haskell.Exts
+import Language.Haskell.Exts hiding (style)
 import Language.Haskell.TH.Syntax                           (Lift(..))
 import HFlags
+#ifdef HTML_OUTPUT
+import Prelude hiding (head, id, div, span)
+import Text.Blaze.Html4.Strict hiding (map, style)
+import Text.Blaze.Html4.Strict.Attributes hiding (title, span)
+import Text.Blaze.Renderer.Utf8 (renderMarkup)
+#endif
 
```

Comment:
You might want to move `out` function here, or into separate module.
Anyone who uses `Message` might be also interested in creating outputs :-).